.github/workflows/ods.yml

```diff
@@ -0,0 +1,36 @@
+name: ODS Build & Test & Publish
+
+on:
+  push:
+    branches:
+    - master
+    - github-ci
+
+jobs:
+  adapter-build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build
+        run: |
+          cd adapter
+          docker build -t docker.pkg.github.com/${{ github.repository }}/adapter .
```

Comment:
Can you also use docker-compose here?

`docker-compose -f docker-compose.yml -f docker-compose.ci.yml build adapter-service`

Reply:
Don't forget to adjust `docker-compose.yml`